packages/calypso-codemods/transforms/sort-imports.js

```diff
@@ -97,7 +96,8 @@ const addNewlineBeforeDocBlock = ( str ) => str.replace( /(import.*\n)(\/\*\*)/,
  * @param {Array} importNodes the import nodes to sort
  * @returns {Array} the sorted set of import nodes
  */
-const sortImports = ( importNodes ) => _.sortBy( importNodes, ( node ) => node.source.value );
+const sortImports = ( importNodes ) =>
+	importNodes.sort( ( nodeA, nodeB ) => nodeA.source.value.localeCompare( nodeB.source.value ) );
```

Comment:
`_.sortBy()` won't mutate the input array, while this one will (since `Array.prototype.sort()` sorts the input array in-place, and returning a reference to the array is a bonus). Should we create a copy of the array before sorting it?

Reply:
Ohh. I missed it. I have created a copy of the array now.